#Success started request
bool started
---
#Success status response
bool status
